src/utils/re_client.py

```diff
@@ -130,6 +130,8 @@ def save(coll_name, docs, on_duplicate='update', display_errors=False):
             'error'.
         display_errors - include error information if insert errors occur.
     """
+    if not isinstance(docs, list) and isinstance(docs, dict):
```

Comment:
Isn't the first check redundant? If it's a dict it's not a list

Reply:
Updated